src/shared/components/MultiFileSelect/MultiFileSelect.tsx

```diff
@@ -206,13 +206,12 @@ export const MultiFileSelect: React.FC<MultiFileSelectProps> = ({
         <Step
           variant="file"
           number={1}
-          title={files.video ? 'Video file' : 'Add video file'}
-          active={step === 'video'}
+          title={files.video ? (files.video.blob ? (files.video.blob as File).name : 'Video file') : 'Add video file'}
```

Comment:
```suggestion
          title={files.video ? (files.video.blob as File).name || 'Video file' : 'Add video file'}
```
Don't know, I think it's more readable and should do the same.

Reply:
Hmm, maybe after all I'm wrong here, so you can ignore this suggestion. But It still not working great, if you open the sheet to edit a video, you'll see: 
![image](https://user-images.githubusercontent.com/51168865/133664107-2f43a722-0b80-41fe-a281-f06861127cbf.png)
 This could be misleading. Can we change this?

Reply:
You mean this is a problem with design or it shouldn't be disabled?

Reply:
no, I think we should display here `Video file`, not `Add video file`